modules/data/src/actions/entity-action-operators.ts

```diff
@@ -18,29 +19,29 @@ import { flattenArgs } from '../utils/utilities';
  */
 export function ofEntityOp<T extends EntityAction>(
   allowedOps: string[] | EntityOp[]
-): OperatorFunction<EntityAction, T>;
+): OperatorFunction<Action, T>;
 export function ofEntityOp<T extends EntityAction>(
   ...allowedOps: (string | EntityOp)[]
-): OperatorFunction<EntityAction, T>;
+): OperatorFunction<Action, T>;
 export function ofEntityOp<T extends EntityAction>(
   ...allowedEntityOps: any[]
-): OperatorFunction<EntityAction, T> {
+): OperatorFunction<Action, T> {
   const ops: string[] = flattenArgs(allowedEntityOps);
   switch (ops.length) {
     case 0:
       return filter(
-        (action: EntityAction): action is T =>
+        (action: any): action is T =>
```

Comment:
Why can't this remain an `EntityAction` as its internal to the function?

Reply:
Because the compiler gives an error if I do so. We now have an `Action` as parameter but this type does not contain the `payload` property. So, the only choice left is to use the `any` type.

Reply:
I don't understand the use case. I would be extremely reluctant to endorse such a change.